src/GoogleAuthenticator.php

```diff
@@ -41,47 +41,62 @@ final class GoogleAuthenticator implements GoogleAuthenticatorInterface
     /**
      * @var int
      */
-    private $codePeriod = 30;
+    private $codePeriod;
+
+    /**
+     * @var int
+     */
+    private $periodSize = 30;
 
     /**
      * @param int                     $passCodeLength
      * @param int                     $secretLength
+     * @param int                     $codePeriod
```

Comment:
We are planning to remove tautological comments such as this one and the ones above, let's not add more of them
```suggestion
```